packages/Popover/src/Popover.js

```diff
@@ -274,7 +291,7 @@ class Popover extends React.Component {
     //       find the first focusable element like button, input, etc?
     //       can focus automatically
     //       should we set focus into the popover content automatically?
```

Comment:
this comment don't make sense anymore, I think we already know what we should do with the focus right? maybe update it or remove the comments?